### PR TITLE
chore: Deprecate document of `askUser`

### PR DESCRIPTION
**Pull request by CsCherrYY**
we now deprecate the documentation of `askUser` from configuration `java.project.exportJar.targetPath`, and the next step will be removing it totally.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -210,13 +210,10 @@
...
               "enum": [
                 "${workspaceFolder}/${workspaceFolderBasename}.jar",
-                "askUser",
                 ""
               ],
               "enumDescriptions": [
-                "%configuration.java.project.exportJar.targetPath.workspaceFolder%",
-                "%configuration.java.project.exportJar.targetPath.askUser%",
-                "%configuration.java.project.exportJar.targetPath.askUser%"
+                "%configuration.java.project.exportJar.targetPath.workspaceFolder%"
```
Comment (jdneo) [inline package.json:216]: Add a description for `""` as well?

Comment (CsCherrYY) [reply package.json]: OK, added the description in https://github.com/microsoft/vscode-java-dependency/pull/575/commits/f1a42c2b5be3f895ab547cc9d24b8e206f220c5b

**Comment by jdneo** [comment]
One more question, where can I see the description for the enums?

image

**Comment by CsCherrYY** [comment]
> One more question, where can I see the description for the enums?
> 
> image

when you're editing this setting in raw settings.json

image

File: package.nls.json [hunk 2]
```diff
@@ -28,11 +28,11 @@
   "configuration.java.dependency.packagePresentation": "Package presentation mode: flat or hierarchical",
   "configuration.java.project.exportJar.targetPath": "Specifies the output path of export jar. The default value is `${workspaceFolder}/${workspaceFolderBasename}.jar`. You can leave it empty to select the output location manually each time exporting the jar file.",
   "configuration.java.project.exportJar.targetPath.workspaceFolder": "Export the jar file into the workspace folder. Its name is the same as the folder's.",
-  "configuration.java.project.exportJar.targetPath.select": "You will select the output location manually each time exporting the jar file.",
+  "configuration.java.project.exportJar.targetPath.select": "Select output location manually when exporting the jar file.",
```
Comment (jdneo) [inline package.nls.json:31]: how about:
```
"configuration.java.project.exportJar.targetPath.customization": "The output path of the exported jar. Leave it to empty if you want to manually pick the output location."
```

Comment (jdneo) [reply package.nls.json:31]: My bad. It should be `leave it empty` instead of `leave it to empty`

**Review by jdneo — APPROVED** [review]

**Pull request merged**